<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1254</width>
    <height>778</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,10">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5,0,2,1,0,0,0,0">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Image Directory:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtDir"/>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Count:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtCount"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnDir">
        <property name="text">
         <string>Confirm</string>
        </property>
        <property name="shortcut">
         <string>Shift+Return</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Finished: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelCount">
        <property name="text">
         <string>Finished</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Deleted: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDelete">
        <property name="text">
         <string>Deleted</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,1">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="img_4">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="img_0">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="img_1">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="img_5">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="img_2">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="img_6">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="img_3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="img_7">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Age:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelAge">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="text_age">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Gender:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelGender">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="text_gender"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Ctrl+s</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_saveResult">
          <property name="text">
           <string>SaveResult</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+S</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Ctrl+d</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_deleteDir">
          <property name="text">
           <string>Delete Directory</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+D</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Ctrl+o</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_openDir">
          <property name="text">
           <string>Open Directory</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+O</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Ctrl+n</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_nextDir">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Next Directory</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+N</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Ctrl+F</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_preDir">
          <property name="text">
           <string>Former Directory</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+F</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1254</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionChoseImageDir">
   <property name="text">
    <string>ChoseImageDir</string>
   </property>
  </action>
  <action name="action_Image_Directory">
   <property name="text">
    <string>Image Directory</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_saveResult</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1501</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>1580</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_deleteDir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1541</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>1587</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_openDir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1530</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>1583</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_nextDir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>nextDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1542</x>
     <y>630</y>
    </hint>
    <hint type="destinationlabel">
     <x>1577</x>
     <y>639</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveResult()</slot>
  <slot>deleteDir()</slot>
  <slot>openDir()</slot>
  <slot>nextDir()</slot>
 </slots>
</ui>
